src/main/java/org/sagebionetworks/web/client/widget/table/modal/fileview/EntityViewScopeWidget.java

```diff
@@ -104,18 +89,8 @@ public void configure(EntityBundle bundle, boolean isEditable) {
 			tableType = TableType.getTableType(currentView);
 			List<Reference> references = getReferencesFromIdList(currentView.getScopeIds());
 
```

Comment:
We were trying to do something funky with two editors--a modal where you could edit the mask and scope, or an inline editor if you could just edit the scope. The inline editor never wrote updates to the backend, so it was impossible to update a project view (or any view which has a mask that you cannot update in the UI).

Instead, always use the modal, and never show the inline edit options.